packages/hooks/src/useAutoScroll/useAutoScroll.story.tsx

```diff
@@ -12,6 +12,7 @@ const testItems = 16;
 
 export default StoryMeta({
   title: 'hooks/useAutoScroll',
+  component: undefined,
```

Comment:
did we double check that the effected stories are still rendering properly on .design?

Reply:
None of the updated ones here are being used on .design